cves/CVE-2012-2779.yml

```diff
@@ -64,7 +64,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: | 
+  The file where the vulnerability existed is of an Indeo Video 
+  Interactive decoder. One function of the video interactive decoder was decoding 
+  a group of pictures (gop) header. The vulnerability was that there was no check 
+  regarding the validity of the gop header being decoded.
```

Comment:
Then what happened? What is the failure that was observed? What could an attacker potentially do?